generator.yaml

```diff
@@ -466,6 +466,47 @@ resources:
         from:
           operation: DescribeMonitoringSchedule
           path: LastMonitoringExecutionSummary
+  FeatureGroup:
+    exceptions:
+      errors:
+          404:
+            code: ResourceNotFound
+            message_prefix: Resource Not Found
+      terminal_codes:
+        - ResourceInUse
+        - ResourceLimitExceeded
+        - ResourceNotFound
+        - InvalidAction
+        - InvalidClientTokenId
+        - InvalidParameterCombination
+        - InvalidParameterValue
+        - InvalidQueryParameter
+        - MalformedQueryString
+        - MissingAction
+        - MissingParameter
+        - OptInRequired
+        - AccessDeniedException
+        - IncompleteSignature
+        - MissingAuthenticationToken
+        - NotAuthorized
+        - ValidationError
+    hooks:
+      delta_pre_compare:
+        code: customSetDefaults(a, b)
+    fields:
+      FailureReason:
+        is_read_only: true
+        is_printable: true
```

Comment:
update code gen and use new format https://github.com/aws-controllers-k8s/code-generator/pull/82